include/vulkan/layer/vk_layer_settings.h

```diff
@@ -21,44 +21,178 @@
 
 #pragma once
 
-#include <string>
-#include <vector>
-
 #include <vulkan/vk_layer.h>
 #include <vulkan/vulkan.h>
```

Comment:
Do we need to expose both? I think just `vk_layer.h` is enough for the interface.

Reply:
Fixed